src/My.Extensions.Localization.Json/Internal/JsonResourceManager.cs

```diff
@@ -178,7 +178,7 @@ IEnumerable<string> GetResourceFiles(string culture)
             if (File.Exists(filePath))
             {
                 var builder = new ConfigurationBuilder()
-                    .AddJsonFile(filePath, optional: true, reloadOnChange: true);
+                    .AddJsonFile(filePath, optional: true, reloadOnChange: false);
```

Comment:
This change is dangerous, so whever your change the JSON file it will not reload the changes